web-client/views/upload.pug

```diff
@@ -513,7 +513,7 @@ html
             p(id='error')
           div(class='modal-footer')
             input(type='button' class='btn btn-default' data-dismiss='modal' value='Close')
-            input(type='button' id='launchFileOpen' class='btn btn-primary' data-dismiss='modal' value='Select New File' onclick="$('.upload').click()")
+            input(type='button' id='launchFileOpen' class='btn btn-primary' data-dismiss='modal' value='Select New File' onclick="$('#upload-network').click()")
```

Comment:
Please see if this works:
```suggestion
            input(type='button' id='launchFileOpen' class='btn btn-primary upload' data-dismiss='modal' value='Select New File')
```

…if this doesn’t work, the general idea is to group this button alongside all of the other controls that trigger the upload handler. That way, we can specify with a single line of code that all of these elements are to invoke the upload handler when clicked/changed

Reply:
I tried to give it an upload class, but it doesn't work. However, if we refer to just one of the upload buttons, it will trigger the uploadHandler.

Reply:
OK, sounds good—yes that will work; the core problem was the `$('.upload')` selector so as long as we target just one element then we’ll be OK